guides/models/finding-records.md

```diff
@@ -82,7 +82,7 @@ For example, if your server API provides an endpoint for the currently logged in
 }
 ```
 
-And if the adapter for the `User` model defines a `queryRecord()` method that targets that endpoint:
+Et si l'adapteur pour le modèle `User` définit une mérhode `queryRecord()` qui cible le `endpoint`:
```

Comment:
1. Translation too word-to-word.
2. No need for code fences for a simple non-code word. Maybe in italic then.

```suggestion
Et si l'adapteur du modèle `User` définit une méthode `queryRecord()` qui cible l' endpoint:
```